src/core_plugins/kbn_vislib_vis_types/public/editors/__tests__/point_series.js

```diff
@@ -136,4 +136,26 @@ describe('point series editor', function () {
     $parentScope.updateAxisTitle();
     expect($parentScope.editorState.params.valueAxes[0].title.text).to.equal('Custom Title');
   });
+
+  it('should set the custom title to match the value axis label when only one agg exists for that axis', function () {
+    $parentScope.editorState.aggs[0].params.customLabel = 'Custom Label';
+    $parentScope.updateAxisTitle();
+    expect($parentScope.editorState.params.valueAxes[0].title.text).to.equal('Custom Label');
+  });
+
+  it('should not set the custom title to match the value axis label when more than one agg exists for that axis', function () {
+    const aggConfig = new AggConfig($parentScope.vis.aggs, { type: 'avg', schema: 'metric', params: { field: 'bytes' } });
+    $parentScope.vis.aggs.push(aggConfig);
+    $parentScope.$digest();
+    $parentScope.editorState.aggs[0].params.customLabel = 'Custom Label';
+    $parentScope.updateAxisTitle();
+    expect($parentScope.editorState.params.valueAxes[0].title.text).to.equal('Count');
+  });
+
```

Comment:
Added the above 2 tests to validate existing editor behavior which was previously untested